### PR TITLE
Custom Donut Chart - Matches pf.org -> Patterns -> Charts -> Donut Chart

### PR DESCRIPTION
**Pull request by dtaylor113**
Update to charts.less and charts test page

### CODE REVIEW

File: less/charts.less [hunk 1]
```diff
@@ -69,3 +69,22 @@
   stroke-dasharray: 0 0;
 }
 
+.donut-tooltip-pf {
```
Comment (rhamilto) [inline less/charts.less]: Please update these rules to follow https://github.com/patternfly/patternfly#coding-style.   You need to alphabetize rules by selector and declarations by property.

File: less/charts.less [hunk 2]
```diff
@@ -69,3 +69,22 @@
...
+  background: #434343;
```
Comment (rhamilto) [inline less/charts.less]: Should use Less variable.  https://github.com/patternfly/patternfly/blob/master/less/variables.less#L202

File: less/charts.less [hunk 3]
```diff
@@ -69,3 +69,22 @@
...
+  color: #fff;
```
Comment (rhamilto) [inline less/charts.less]: Should use Less variable.  https://github.com/twbs/bootstrap/blob/master/less/variables.less#L523

File: less/charts.less [hunk 4]
```diff
@@ -69,3 +69,22 @@
...
+  opacity: 0.9;
+  filter: alpha(opacity=90);
+  padding: 2px 6px;
+}
+
+.donut-title-big-pf {
+  font-size: 30px;
+  font-weight: 300;
+  color: #333;
```
Comment (rhamilto) [inline less/charts.less]: Should use Less variable.  https://github.com/patternfly/patternfly/blob/master/less/variables.less#L164

File: less/charts.less [hunk 5]
```diff
@@ -69,3 +69,22 @@
...
+}
+
+.donut-title-small-pf {
+  font-size: 12px;
+  font-weight: 400;
+  color: #333;
```
Comment (rhamilto) [inline less/charts.less]: Should use Less variable. https://github.com/patternfly/patternfly/blob/master/less/variables.less#L164

File: less/charts.less [hunk 6]
```diff
@@ -69,3 +69,22 @@
...
+}
+
+.donut-title-big-pf {
+  font-size: 30px;
+  font-weight: 300;
+  color: #333;
+}
+
+.donut-title-small-pf {
+  font-size: 12px;
```
Comment (rhamilto) [inline less/charts.less]: Should use Less variable.  https://github.com/patternfly/patternfly/blob/master/less/variables.less#L163

File: less/charts.less [hunk 7]
```diff
@@ -69,3 +69,22 @@
...
+.donut-tooltip-pf {
+  background: #434343;
+  color: #fff;
+  opacity: 0.9;
+  filter: alpha(opacity=90);
+  padding: 2px 6px;
+}
+
+.donut-title-big-pf {
+  font-size: 30px;
```
Comment (rhamilto) [inline less/charts.less]: Should use a Less variable that you need to define.

File: tests/charts.html [hunk 8]
```diff
@@ -65,6 +65,8 @@
       <div id="chart7"></div>
       <h2>Donut Chart</h2>
       <div id="chart8"></div>
+      <h2>Custom Donut Chart</h2>
```
Comment (rhamilto) [inline tests/charts.html:68]: Per https://github.com/patternfly/patternfly#tests, these changes should be made in tests-src/charts.html as it is the source for this file.

**Comment by dtaylor113** [comment]
All issues addressed
Thanks!
- Dave

File: tests/charts.html (hunk 8 above)
Comment (rhamilto) [inline tests/charts.html:68]: Additionally, you need to run `grunt server` to compile and view the Less to CSS and generate the test pages or at least `grunt build` to compile and generate.

File: less/variables.less [hunk 9]
```diff
@@ -90,6 +90,7 @@
 @table-border-hover:                                                #a7cadf;
 @table-cell-padding-bottom:                                         3px;
 @table-cell-padding-top:                                            2px;
+@donut-font-size-big:                                               30px;
```
Comment (rhamilto) [inline less/variables.less]: Please alphabetize this variable within this section (move it to line 19).

**Comment by rhamilto** [comment]
As a reminder, you should always work of of a feature branch instead of master when submitting PRs.

File: tests/charts.html (hunk 8 above)
Comment (dtaylor113) [inline tests/charts.html:68]: Ok, I've been doing that.  Are you saying I need to check in the dist files?

Comment (rhamilto) [inline tests/charts.html:68]: Yes.

**Comment by dtaylor113** [comment]
ok, all issues addressed

**Comment by rhamilto** [comment]
LGTM.  @andresgalante or @erundle?

File: tests/charts.html [hunk 10]
```diff
@@ -379,6 +381,50 @@
...
+            columns: [
+              ["Used", 60],
+              ["Available", 40]
+            ],
+            groups: [
+              ["used", "available"]
+            ],
+            order: null
+          },
+        });
```
Comment (erundle) [inline tests/charts.html:419]: I get this is just example code but in the future I would like to more towards encapsulating this configuration.

Comment (dtaylor113) [inline tests/charts.html:419]: Can you please elaborate on this?

**Pull request merged**